### ISSUE TITLE
Update link to documentation 

### ISSUE THREAD

**Issue by esthersweon** [regl-project/regl#537]
Github page still cites `http://regl.party` as link to documentation, which now seems outdated.

**Comment by rreusser** [comment]
@mikolalysenko I don't think anyone has the credentials to deploy the site. Would you mind re-deploying? Otherwise I'd be glad to take some ownership over website maintenance or enabling others to do the same. Thanks!

**Comment by rreusser** [comment]
@esthersweon It should definitely be updated, but in the mean time, I always rely on either https://github.com/regl-project/regl/blob/master/API.md for API docs or http://regl-project.github.io/regal/www/gallery.html for the example.

**Comment by lostfictions** [comment]
If someone has "maintain" or "admin" permissions for the repo maybe they could at least remove the url from the repo description until the site can be redeployed? That'd probably save some confusion...

**Comment by rreusser** [comment]
Good idea, @lostfictions. I've removed it from the github repo description line. Seems better if the repo speaks for itself. 👍

**Comment by mootari** [comment]
FYI, http://regl.party is back online. The old Surge host was still running, and I've secured the expired domain for the next five years.
@rreusser Please let me know if you plan to redeploy the site to a different IP.

**Comment by rreusser** [comment]
Thanks, @mootari! It's been a bit of a pending item for quite a while to get some changes/fixes deployed to the site. I think the missing link has been the surge.sh credentials. If you want the truth, I'm not against just hosting on github pages since it's sorta default-maintainable-and-online. Don't have the time right at the moment to work on a site refresh, though I wouldn't be against it since I think we could certainly work toward adding more information, example, and tutorials.

**Comment by rreusser** [comment]
Time flies! 😱 TBH my first preference is to just use the fail-safe (fail-perfectly-adequate?) github pages link. Especially with the advent of WebGPU and wide adoption of WebGL 2, I don't think regl is irrelevant (I still use it all the time), but I do think it has a somewhat finite lifetime. If you think it's better to avoid breaking links then we can renew and I'd be glad to adopt it. (FWIW I just use https://github.com/regl-project/regl/blob/master/API.md and haven't ever really used the regl.party site 🤷 )

**Comment by mootari** [comment]
While I don't have a strong opinion either way, I think my recommendation would be to keep the domain around for another year but either redirect it or take it offline, so that search engines can purge any results and no one will be able to grab and "repurpose" the domain.

If that's the path we want to take I'm happy to drag the domain along for another year. 🙂

**Comment by rreusser** [comment]
Sounds good. Okay, I've renamed [regl-site](https://github.com/regl-project/regl-project.github.io) and connected it to github pages: https://regl-project.github.io/

I've also disconnected the site from any of the JavaScript app code and redirected the links to static pages and the github version of the API docs. If we can add a CNAME redirect from regl.party to regl-project.github.io, then maybe it will be fine to drop the domain after another year or two. Kinda cowboying the maintenance tasks here, but my main goal is to just get things into a fail-safe state. Let me know what you think.

**Comment by rreusser** [comment]
It's not perfect (not a live example with editor 🤷), but I've also redirected the main regl README from regl.party to the github pages site.

**Comment by mootari** [comment]
I've renewed the domain for another year and added a wildcard HTTP redirect to https://regl-project.github.io/ which should become active within ~30 minutes.

**Comment by mootari** [comment]
> Unfortunately, these will never automatically get purged, and I don't have particular interest in forking and PRing tens of links in decade-old projects.

That's fair. I'll check if I can produce a list that ranks those results by stars, so that we can at least get the most popular ones updated (likely a bunch of awesome-* lists).

**Comment by mootari** [comment]
> I'll check if I can produce a list that ranks those results by stars, so that we can at least get the most popular ones updated

I put up a list of repos and their stargazers counts at https://observablehq.com/d/3d810e7c9970553c
